src/NuGet.Core/NuGet.Commands/RestoreCommand/CompatibilityChecker.cs

```diff
@@ -236,7 +251,7 @@ private static RestoreLogMessage GetErrorMessage(NuGetLogCode logCode, Compatibi
                 graph.Conventions.Patterns.ResourceAssemblies,
                 graph.Conventions.Patterns.CompileRefAssemblies,
                 graph.Conventions.Patterns.RuntimeAssemblies,
-                graph.Conventions.Patterns.ContentFiles
+                graph.Conventions.Patterns.ContentFiles,
```

Comment:
why the , in the end?

Reply:
Stuff was added, then removed in a later commit, but the , got left over.